app/controllers/user.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const bcrypt = require('bcrypt'),
+  saltRounds = 10,
+  salt = bcrypt.genSaltSync(saltRounds);
+
+const User = require('../models').users;
+
+const validateEmail = valor => {
+  if (/^\w+([\.-]?\w+)@wolox+(\.\w{2,3})+$/.test(valor)) {
```

Comment:
Return condition instead of bool, `return /^\w+([\.-]?\w+)@wolox+(\.\w{2,3})+$/.test(valor)`